src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PENamedTypeSymbol.cs

```diff
@@ -504,7 +505,7 @@ private ImmutableArray<NamedTypeSymbol> MakeDeclaredInterfaces()
                         TypeSymbol typeSymbol = tokenDecoder.GetTypeOfToken(interfaceHandle);
 
                         typeSymbol = TupleTypeDecoder.DecodeTupleTypesIfApplicable(typeSymbol, interfaceImpl, moduleSymbol);
-                        typeSymbol = NullableTypeDecoder.TransformType(TypeWithAnnotations.Create(typeSymbol), interfaceImpl, moduleSymbol).Type;
+                        typeSymbol = NullableTypeDecoder.TransformType(TypeWithAnnotations.Create(typeSymbol), interfaceImpl, moduleSymbol, this, GetNullableContextValue()).Type;
```

Comment:
>GetNullableContextValue() [](start = 146, length = 25)

Can we request ```GetNullableContextValue()``` lazily only when needed inside ```NullableTypeDecoder.TransformType``` instead calculating this eagerly on every invocation of ```NullableTypeDecoder.TransformType```?